services/payments/controllers.go

```diff
@@ -84,7 +105,7 @@ func PrepareHandler(service *Service) handlers.AppHandler {
 		}
 
 		// returns an enriched list of transactions, which includes the document metadata
-		resp, err := service.PrepareTransaction(ctx, req)
+		resp, err := service.PrepareTransaction(ctx, &namespaceUUID, req)
```

Comment:
Is there a reason we use a pointer here opposed to the value?

Reply:
No I think it's just a holdover from the prior structure. Fixed.